index.html

```diff
@@ -12,19 +12,24 @@
 
 <body>
   <main>
+  
     <header class="header-class">
       <nav class="nav-container">
         <a href="#" class="flex-logo"><img src="./img/My-Logo.png" alt="logo"></a>
         <div class="flex-menu"><img src="./img/Menu-button.png" alt="menu"></div>
+        <ul class="desktop-toolbar">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
       </nav>
     </header>
-
-    <section class="headline-section">
+    <section class="headline-section headline-sectionds">
```

Comment:
This Section it's overlapping your application in your X-Axis Kindly fix it.
Your live version:
![Screen Shot 2021-08-27 at 20 50 39](https://user-images.githubusercontent.com/22258274/131202514-6330ef28-7f1d-4f30-8565-19f7d2dc50d9.png)
Local Version:
![Screen Shot 2021-08-27 at 20 52 35](https://user-images.githubusercontent.com/22258274/131202544-85d1151c-15ec-49c7-bc7b-7b427ed98088.png)

Reply:
Thank you for your constructive review, but I didn't really understand. Can't you explain more please.